src/API.jl

```diff
@@ -940,6 +938,7 @@ function precompile(ctx::Context)
         was_recompiled[pkgid] = false
     end
     
+    msg_printed = false
```

Comment:
There is no race condition to worry about here if multiple tasks modify this value?

Reply:
At this point the tasks are all async, so on the same thread. So I don’t believe so

Reply:
I realized I can just check for `!any(values(was_recompiled))` so I removed `msg_printed`. Should be just as ok, with less code